src/applications/vaos/new-appointment/components/TypeOfCarePage/TypeOfCareAlert.jsx

```diff
@@ -8,6 +8,7 @@ export default function TypeOfCareAlert() {
     <AlertBox
       status="info"
       headline={headline}
+      level="2"
```

Comment:
Looking at our other AlertBox(es), they don't have "level" attribute in the component.  Did you find the recommended attribute from the Axe scan? I can't seem to duplicate the result nor find the rule. Just curious if we need to add "level" to all the other AlertBox(es).

Reply:
If you run an aXe scan on http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-appointment (without the change in this PR), you should get a moderate issue that says "Heading levels should only increase by one." That's happening because the other heading on this page is an `h1` and the alert box by default has an `h3`.

The `AlertBox` [component](https://design.va.gov/storybook/?path=/docs/components-alertbox--info) has a `level` prop that allows you to adjust that `h3` to fit the context of your page.

There could be other places where we need to set the level for the `AlertBox`, but it depends on the context of the page.